development.ini

```diff
@@ -47,21 +47,25 @@ jujugui.interactive_login = true
 # Default  URLS
 jujugui.charmstore_url = https://api.staging.jujucharms.com/charmstore/
 jujugui.plans_url =
+jujugui.register_user_url =
 jujugui.terms_url =
 
 # Staging URLS
 ;jujugui.charmstore_url = https://api.staging.jujucharms.com/charmstore/
 ;jujugui.plans_url = https://api.staging.jujucharms.com/omnibus/
+;jujugui.register_user_url = https://api.staging.jujucharms.com/payment/
 ;jujugui.terms_url = https://api.staging.jujucharms.com/terms/
 
 # Production
 ;jujugui.charmstore_url = https://api.jujucharms.com/charmstore/
 ;jujugui.plans_url = https://api.jujucharms.com/omnibus/
+;jujugui.register_user_url = https://api.jujucharms.com/payment/
 ;jujugui.terms_url = https://api.jujucharms.com/terms/
 
 # jujugui.org
 ;jujugui.charmstore_url = https://api.jujugui.org/charmstore/
 ;jujugui.plans_url = https://api.jujugui.org/omnibus/
+;jujugui.register_user_url = https://api.jujugui.org/payment/
```

Comment:
Those jujugui.org URLs are all wrong. While you're here, please replace api.jujugui.org with www.jujugui.org everywhere.